dev-env/windows/manifests/7zip.json

```diff
@@ -1,21 +1,41 @@
 {
-  "homepage": "http://www.7-zip.org/",
-  "description": "A multi-format file archiver with high compression ratios",
-  "license": {
-    "identifier": "LGPL-2.0-only,BSD-3-Clause",
-    "url": "https://www.7-zip.org/license.txt"
-  },
-  "version": "18.05",
-  "architecture": {
-    "64bit": {
-      "url": "https://www.7-zip.org/a/7z1805-x64.msi",
-      "hash": "898c1ca0015183fe2ba7d55cacf0a1dea35e873bf3f8090f362a6288c6ef08d7"
+    "homepage": "https://www.7-zip.org/",
```

Comment:
We use 2-space indentation for all the files in this directory. Might be worth staying consistent.

Reply:
I copied this from scoop upstream, I’d rather not make unnecessary modifications.